helm/app/templates/dashboard-role.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Same here
i think we should add if wrapping whole chart when kubernetes-dashboard is enabled


```
{{ if or (not (hasKey .Values.kubernetes-dashboard "enabled")) (.Values.kubernetes-dashboard.enabled | default true) }}
...
{{- end }}
```

I dont think we need this role if we disable the chart correct ?
https://github.com/vardius/go-api-boilerplate/wiki/2.1.-Dashboard#turn-off